<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Trans</class>
 <widget class="QDialog" name="Trans">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1px solid white;
border-radius: 10px;
background: white;

background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgba(72, 145, 217, 255), stop: 1 rgba(255, 255, 255, 255));
</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Crud</string>
    </attribute>
    <widget class="QLabel" name="label_mat">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Matricule </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_Mat">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre des voyageurs</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_nbV">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_type">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Type de véhicule</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_type">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Bus</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Voiture </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mini_Bus</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat Véhicule</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_etat">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>190</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Bonne</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EnPanne</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="Ajouter">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Modifier">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_r">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>330</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>370</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recherche">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>370</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Trier">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>220</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>trie</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>411</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_trie">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>220</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>MATRICULE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NB VOYAGEURS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TYPE</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pdf">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>270</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Pdf</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stat">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>240</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Statistique </string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_r">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>270</y>
       <width>381</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Mailing</string>
    </attribute>
    <widget class="QLineEdit" name="rcpt">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>recipient</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mail_pass">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>20</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>your mail password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="subject">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>subject</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>421</width>
       <height>23</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>attachement here</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="msg">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>741</width>
       <height>231</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>type your message here ...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendBtn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>400</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browseBtn">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>90</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse ...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Map</string>
    </attribute>
    <widget class="WebAxWidget" name="WebBrowser">
     <property name="control" stdset="0">
      <string>{8856F961-340A-11D0-A96B-00C04FD705A2}</string>
     </property>
     <property name="focusPolicy" stdset="0">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="geometry" stdset="0">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>691</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
  <customwidget>
   <class>WebAxWidget</class>
   <extends>QAxWidget</extends>
   <header>webaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
